charts/scalardb-mcp-server/templates/scalardb-mcp-server/ingress.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "scalardb-mcp-server.fullname" . -}}
+{{- $svcPort := .Values.scalardbMcpServer.service.port -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
+  {{- end }}
+{{- end }}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
```

Comment:
Basically, we support the maintained version of Kubernetes, and we support from `1.31` to `1.34` as mentioned in our official document.

- Requirements doc
  - https://scalardb.scalar-labs.com/docs/latest/requirements/#platform

Therefore, I think we need to take care of the older (already not supported) versions.

Do we realy need this if statement here?